docs/README.md

```diff
@@ -13,6 +13,42 @@ This repo consists of 2 packages:
 
 If you are using Node, you should use our [`bitski-node`](https://github.com/BitskiCo/bitski-node) package instead.
 
+### Example App
+
+Want to see Bitski in action? You can check out a [full demo app using our SDK](https://example-dapp-1.bitski.com) and [view the demo's source code](https://github.com/BitskiCo/example-dapp-game).
+
+### Basic Usage
+
+A very simple integration of the Bitski would look something like this.
+
+```javascript
+import { Bitski } from 'bitski';
+import Web3 from 'web3';
+
+const bitski = new Bitski('CLIENT-ID', 'https://myapp.com/callback.html');
+
+const provider = bitski.getProvider();
+const web3 = new Web3(provider);
+
+// public calls are always available
+const network = await web3.eth.getBlockNumber();
+
+// connect via oauth to use the wallet
+await bitski.start();
```

Comment:
Does this need a user action to call?

Reply:
Might want to put that in a comment.

Reply:
In some cases yes, but this example is more pseudo code than copy / paste. The reality is that no ones code looks remotely like this and demonstrating a click handler here makes it much harder to follow.